### ISSUE TITLE
Broken CachedImage scaling on Android

### ISSUE THREAD

**Issue by rafalka** [microspaze/FFImageLoading.Maui#22]
### Description

Changes introduced in version 1.0.8 ( I guess that by [this commit](https://github.com/microspaze/FFImageLoading.Maui/commit/e1cfad81a5f825fb487fead5a1ad7b6835a85cb8?diff=split&w=0) )
broke  CachedImage scaling on Android.
iOS version is fine.

### Steps to Reproduce
This basic page from attached project:
```XAML
    <Grid ColumnDefinitions="*" RowDefinitions="Auto,Auto,Auto,Auto,Auto">
        <Label Grid.Row="0"  Text="FFImageLoading image:"/>
        <ffimageloading:CachedImage Grid.Row="1"
                                    HorizontalOptions="Center" VerticalOptions="Center"
                                    Source = "no_photo.png">
        </ffimageloading:CachedImage>
        <Label Grid.Row="2"  Text="Should keep its original size"/>
        <Image Grid.Row="3"
               Source="no_photo.png"/>
        <Label Grid.Row="4"  Text="As it is on Xamarin/MAUI image"/>
```
### Expected Behavior
should look like this:
![image](https://github.com/microspaze/FFImageLoading.Maui/assets/3198039/3dd2136b-66d4-4a05-a247-b41841030c4b)

### Actual Behavior
but it looks like this:
![image](https://github.com/microspaze/FFImageLoading.Maui/assets/3198039/5a4132bb-2b93-435e-8af8-3d8dbed5ceec)

### Basic Information

- Version with issue: >= 1.0.8
- Last known good version: 1.0.7
- Platform: Android

### Reproduction Link / Code

I'm attaching two sample project: 
`FFImageLoadingBug.Maui` - for this bug reproduction
`FFImageLoadingBug.Xamarin` - for comparison with original [FFImageLoading](https://github.com/luberda-molinet/FFImageLoading)

[FFImageLoadingBug.zip](https://github.com/microspaze/FFImageLoading.Maui/files/14834825/FFImageLoadingBug.zip)

**Comment by microspaze** [comment]
I will refine the old commit to fix this bug tomorrow. Thanks for reporting!

**Comment by rafalka** [comment]
Thanks!

BTW, In meantime I've checked the  MAUI `ImageHandler.Android.cs` 
In `CreatePlatformView()`
they call `platformView.SetAdjustViewBounds(true);` and this does the trick. 
Overriding `GetDesiredSize()` is no longer needed. Width and height request are handled properly.
If you'll find some time then you can explore this approach

**Comment by microspaze** [comment]
Fixed in the latest version 1.1.5  https://www.nuget.org/packages/FFImageLoading.Maui/1.1.5

Thanks reporting.

**Comment by rafalka** [comment]
Thanks @microspaze for fixing this!
